templates/provider/cluster-api-provider-openstack/templates/orc-2-1-0.yaml

```diff
@@ -5117,7 +5117,7 @@ spec:
         - --health-probe-bind-address=:8081
         command:
         - /manager
-        image: quay.io/orc/openstack-resource-controller:v2.1.0
+        image: {{ default "quay.io" (and .Values.global .Values.global.registry) }}/orc/openstack-resource-controller:v2.1.0
```

Comment:
This will be overwritten by any next `make capo-orc-fetch` call with an updated `CAPO_VERSION`. I think we should fix it within the recipe itself. Also, does it make sense to partially hardcode the rest of the path (along with the image's version) in case of `global.registry`?

Extra: I think now we could safely drop the `orc-1-0-0.yaml` and stop versioning the file name since only we control this manifest.